app/src/main/assets/drawable.xml

```diff
@@ -5654,4 +5654,5 @@
 	<item component="ComponentInfo{com.insideoutside.ihunteron/com.insideoutside.ihunter.iHunterActivity}" drawable="ihunter" />
 	<item component="ComponentInfo{au.com.redcat.chatime/au.com.redcat.chatime.MainActivity}" drawable="chatime" />
 	<item component="ComponentInfo{com.dhb.microrpg/com.unity3d.player.UnityPlayerActivity}" drawable="micro_rpg" />
+	<item component="ComponentInfo{com.crazylegend.vigilante/com.crazylegend.vigilante.MainActivity}" drawable="vigilante" />
```

Comment:
This line doesn ot belong in drawable.xml. Sorry if the above lines are confusing you, we'll see that those are removed or fixed.

Reply:
so i just remove it?